operators/hack/deployer/cmd/create.go

```diff
@@ -0,0 +1,57 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+const defaultRunConfigTemplate = `id: gke-dev
+overrides:
+  clusterName: %s-dev-cluster
+  gke:
+    gCloudProject: %s
+`
+
+func CreateCommand() *cobra.Command{
+	var path string
+
+	var createCommand = &cobra.Command{
+		Use:   "create",
+		Short: "Creates run config file.",
+	}
+
+	var createDefaultConfigCommand = &cobra.Command{
+		Use:   "defaultConfig",
+		Short: "Creates default dev config using USER and GCLOUD_PROJECT env variables.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			user, ok := os.LookupEnv("USER")
+			if !ok {
+				return fmt.Errorf("USER environment variable not present")
+			}
+
+			gCloudProject, ok := os.LookupEnv("GCLOUD_PROJECT")
```

Comment:
I have mixed feelings about making GKE the default. Wondering if we should not call for it explicitly: there is no defaults, if you want GKE you ask for GKE.

Reply:
Is adding new set of targets (bootstrap, delete, credentials, set-context) for each provider a value add? The usage pattern I had in mind was to default everything to gke during first run (so that initial run is as simple as possible) and allow user to modify run-config.yml as needed for all commands. So it's kind of opposite direction where commands are generic and driven by config file present. I'm not sure how often we would be switching between aks/gke/eks during development, hence the above. 

I'll do the renames right now and we can discuss to stop defaulting to gke separately. WDYT?

Reply:
Got it. Sounds good. I'm also fine with going forward with what you propose and change it later if needed.